### PR TITLE
Additional info for HSTS configuration

### LINKED ISSUE #2856: Update HSTS Doc
Re: <https://pantheon.io/docs/hsts/>
## Issue Description
(From the SME)
The suggested expiry times vary as if it depends on the module/plugin, and seems to imply that is how it is, but that is a configuration the user should understand/set. also there is no mention of `includeSubDomains` and `preload` which are important parts of the configuration too.

## Suggested Resolution
Address the above issues, and define one year and 6-month expiry numbers:

One year expiry:
`Strict-Transport-Security: max-age=31536000;`

Six month expiry with include subdomains and add to browser preload list:
`Strict-Transport-Security: max-age=15984000; includeSubDomains; preload`

### PR DESCRIPTION
**Pull request by peter-pantheon**
Closes #2856

## Effect
PR includes the following changes:
- Additional resource(s) link(s)
- Additional info on using HSTS headers

## Remaining Work
- [x] Proof for style, shorten

### CODE REVIEW

File: source/_docs/hsts.md [hunk 1]
```diff
@@ -51,3 +71,31 @@ Once installed and configured, the following header will be sent in responses:
 ```http
 strict-transport-security: max-age=15552000
 ```
+</div>
+
+</div>
+
+
+## Configure Attributes
```
Comment (rachelwhitton) [inline source/_docs/hsts.md]: Suggest adding instructions to this section for how to configure attributes. If step by step can't be provided, maybe some general advice for how to do this?

Comment (alexfornuto) [reply source/_docs/hsts.md]: Good idea, I'll see if @peter-pantheon can look into this during his next doc block.

Comment (peter-pantheon) [reply source/_docs/hsts.md]: @rachelwhitton @alexfornuto 

User's can inject the header via a (configurable) WordPress plugin, (configurable)  Drupal module, code or by hard coding the header into a template file. So I think giving general advisement on configuration options is acceptable here.  

I see there is a header called "Configure Atrributes" and info below it and that looks good to me. What more specifics would be desired here? 

There are some details here that may require further offline discussion too,

**Comment by rachelwhitton** [comment]
+1 Looks good, thanks @peter-pantheon and @alexfornuto :shipit:

**Comment by alexfornuto** [comment]
Thanks @peter-pantheon!

**Pull request merged**